desktop/cmp/input/CodeInput.js

```diff
@@ -238,7 +248,7 @@ export class CodeInput extends HoistInput {
             },
             foldGutter: true,
             scrollbarStyle: 'simple',
-            readOnly: disabled ? 'nocursor' : false,
+            readOnly: disabled,
```

Comment:
what's the deal with the 'nocursor' string from before.  Is that still of concern?

Reply:
That was what was prevent John H from selecting the text or focusing the input. It's like an even more extreme version of 'disabled'

Reply:
sounds good